libs/wingc/src/jsify.rs

```diff
@@ -1325,7 +1326,10 @@ fn get_public_symbols(scope: &Scope) -> Vec<Symbol> {
 }
 
 fn inflight_filename(class: &AstClass) -> String {
-	format!("./inflight.{}.js", class.name.name)
+	let mut hasher = std::collections::hash_map::DefaultHasher::new();
```

Comment:
An alternative can be to use the file_id plus a counter to create the file name, something like:
`format!("./inflight.{}${}-{}.js", sanitize(class.name.file_id), inflight_name_counter++, class.name.name)`

Reply:
Yeah, I guess I wanted to try something non-counter based to make `.wing` outputs a little more stable, but it's not critical for sure. Let's try the current implementation / see if it causes any issues